cmd/influxd/config.go

```diff
@@ -60,18 +87,39 @@ var DefaultSnapshotURL = url.URL{
 	Host:   net.JoinHostPort(DefaultSnapshotBindAddress, strconv.Itoa(DefaultSnapshotPort)),
 }
 
+// Broker represents the configuration for a broker node
+type Broker struct {
+	Dir     string   `toml:"dir"`
+	Enabled bool     `toml:"enabled"`
+	Timeout Duration `toml:"election-timeout"`
+}
+
+// Snapshot represents the configuration for a snapshot service
+type Snapshot struct {
```

Comment:
I think it's best as is, but it would be worth commenting the file, letting the user know this only applies to data nodes.